knowlg-automation/terraform/aws/eks_addon.tf

```diff
@@ -0,0 +1,13 @@
+# resource "aws_eks_addon" "addons" {
+#   for_each                 = {for addon in var.eks_addons : addon.name => addon }
+#   cluster_name             = aws_eks_cluster.eks_cluster.id
+#   addon_name               = each.value.name
+#   addon_version            = each.value.version 
+#   resolve_conflicts_on_update = "PRESERVE"
+# }
+resource "aws_eks_addon" "example" {
+  cluster_name                = aws_eks_cluster.eks_cluster.name
+  addon_name                  = "coredns"
```

Comment:
Why did we hardcode this?

Reply:
it was showing as the version outdated as the error so I tried to hardcode it. what could be the possible case to do it while also specifying the latest version (as the previous configuration was not taking the latest)?

Reply:
you are already providing it in the variables.tf - https://github.com/Sunbird-Knowlg/knowledge-platform/blob/25114f698a5f228047cfad532272d6f09ab1606a/knowlg-automation/terraform/aws/variables.tf#L117

Reply:
seems like yeah i need to change it here only then